### PR TITLE
(MODULES-9370) Align module to core structure

### PR DESCRIPTION
**Pull request by florindragos**

### CODE REVIEW

File: spec/spec_helper_acceptance.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 def add_test_user
```
Comment (tphoney) [inline spec/spec_helper_acceptance.rb]: this does not seem to follow the litmus template for spec_helper_acceptance. These helpers should be in spec_helper_acceptance_local.rb

File: spec/unit/puppet/provider/scheduled_task/win32_taskscheduler_spec.rb [hunk 2]
```diff
@@ -20,1586 +20,1553 @@
...
-          },
-          {
-            'start_date'       => '2013-12-15',
-            'start_time'       => '16:23',
-            'schedule'         => 'once',
-            'minutes_interval' => 60,
-            'minutes_duration' => 240,
-            'enabled'          => true,
-            'index'            => 2,
-          }
```
Comment (tphoney) [inline spec/unit/puppet/provider/scheduled_task/win32_taskscheduler_spec.rb]: this block of tests, seems to have dissappeared

Comment (florindragos) [reply spec/unit/puppet/provider/scheduled_task/win32_taskscheduler_spec.rb]: it is identical to the one below

File: spec/spec_helper_acceptance.rb (hunk 1 above)
Comment (ThoughtCrhyme) [reply spec/spec_helper_acceptance.rb]: It is consistent with the documentation https://github.com/puppetlabs/puppet_litmus/wiki/Converting-a-module-to-use-Litmus#specspec_helper_acceptancerb

Comment (ThoughtCrhyme) [reply spec/spec_helper_acceptance.rb]: Also, if I may ask, why are we adding "local" to the end of the file? That seems to imply that it's only used for localhost acceptance testing which isn't the case.

Comment (tphoney) [reply spec/spec_helper_acceptance.rb]: @ThoughtCrhyme If you look at any of the modules, the _local.rb files denote local differences to the templated ruby files. hence https://github.com/puppetlabs/puppetlabs-apt/blob/master/spec/spec_helper_local.rb https://github.com/puppetlabs/puppetlabs-postgresql/blob/master/spec/spec_helper_local.rb for unit tests. https://github.com/puppetlabs/puppetlabs-postgresql/blob/master/spec/spec_helper_acceptance_local.rb and https://github.com/puppetlabs/puppetlabs-apt/blob/master/spec/spec_helper_acceptance_local.rb for non standard functions. If you feel this is unclear please update the wiki. 

The wiki states to use the template provided.

**Comment by michaeltlombardi** [comment]
Reworked and merged in #88

**Pull request closed without merging**